modules/navigation.js

```diff
@@ -0,0 +1,20 @@
+function showContent(section) {
```

Comment:
- [ ] Save here too as pointed out earlier :point_up: Kindly change this function to **arrow** function syntax.